### PR TITLE
[CB-165] Add new row to Search History on Search

### PR DESCRIPTION
**Pull request by LWanjiru**
# JIRA TICKET NAME:
CB-165: [Add new row to Search History on Search](https://issues.openmrs.org/browse/CB-165)

## SUMMARY:
Currently, when a user populates the query form and clicks on search, the results table automatically opens, displaying the patients and their details.

After bug fix, when a user populates the form and clicks on Search, they should remain on the same page. And a new row with the fetched results, should be added to the Search history.

### CODE REVIEW

File: app/js/components/App.jsx [hunk 1]
```diff
@@ -34,26 +35,31 @@ class App extends Component {
     });
   }
 
+  setHistory() {
+    this.setState({ 
+      history : data.rows || data.patients
+    });
+    console.log(history);
```
Comment (mkiterian) [inline app/js/components/App.jsx]: Forgot to take this out

Comment (LWanjiru) [reply app/js/components/App.jsx]: Thank you. Resolved.

File: app/js/cohortbuilder.jsx [hunk 2]
```diff
@@ -10,22 +10,15 @@
 
 import React from 'react';
 import {render} from 'react-dom';
-
 import {Router, Route, hashHistory} from 'react-router';
-import {Provider} from 'react-redux';
+import App from './components/App.jsx'
```
[the remark this answers is not in the document]
Comment (LWanjiru) [reply app/js/cohortbuilder.jsx]: Resolved.

File: app/js/components/App.jsx [hunk 3]
```diff
@@ -34,26 +35,30 @@ class App extends Component {
     });
   }
 
+  setHistory() {
+    this.setState({ 
+      history : data.rows || data.patients
```
[the remark this answers is not in the document]
Comment (LWanjiru) [reply app/js/components/App.jsx]: Resolved.

File: app/js/cohortbuilder.jsx [hunk 4]
```diff
@@ -10,22 +10,15 @@
...
 import React from 'react';
 import {render} from 'react-dom';
-
 import {Router, Route, hashHistory} from 'react-router';
-import {Provider} from 'react-redux';
+import App from './components/App';
 
-import createStore from './redux-store';
-import routes from './routes';
 // css necessary for custom react-select input fields which would be used
```
Comment (CeciliaWanjiku) [inline app/js/cohortbuilder.jsx]: are this comments necessary? 
Otherwise good job

File: app/js/components/App.jsx [hunk 5]
```diff
@@ -34,26 +35,30 @@ class App extends Component {
...
       <div>
           <div 
             id="tabbed-cohort" style={{display}}>
             <Header/>
             <BreadCrumbComponent/>
-            <PageComponent getHistory = {this.getHistory} />
+            <PageComponent getHistory = {this.getHistory} setHistory = {this.setHistory} />
           </div>
-
           <div id="body-wrapper"
```
Comment (jacksono) [inline app/js/components/App.jsx]: This is too outdented

**Comment by jacksono** [comment]
Looks good.

File: app/js/cohortbuilder.jsx (hunk 4 above)
Comment (LWanjiru) [reply app/js/cohortbuilder.jsx]: Thank you for pointing that out.  Resolved.

**Pull request merged**